core/actions/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const config = require('../../config');
```

Comment:
Maybe this change could be done in the image (Dockerfile) with a Symlink ?

This is going to run inboth formats (docker and asg) during a short time, but no easy predictable.

Reply:
I have reverted the change that Guardado pushed and created a symlink in order to be backwards compatible.

I have deployed a new image with this change and it is working fine.